src/directives/map.js

```diff
@@ -75,7 +75,7 @@ ngeo.mapDirective = function(ngeoSyncDataSourcesMap) {
           window,
           'resize',
           () => {
-            if (resizeTransition) {
+            if (resizeTransition && Number.isInteger(resizeTransition)) {
```

Comment:
The compiler complains because we get `resizeTransition` without knowing its type.  You're right, it can't.  Any suggestion?